test/libsolidity/semanticTests/smoke_test.sol

```diff
@@ -0,0 +1,17 @@
+contract C {
+    function f() public returns (uint) {
+        return 1;
+    }
+    function g(uint x, uint y) public returns (uint) {
+        return x - y;
+    }
+    function h() public payable returns (uint) {
+        return f();
+    }
+}
+// ----
+// f() -> 1
+// g(uint256,uint256): 1, -2 -> 3
+// h(), 1 ether -> 1
+// j() -> FAILURE
+// i() # Does not exist. # -> FAILURE # Reverts. #
```

Comment:
Is this correct? Why not `-> FAILURE`?

Reply:
That's what it says I think - The second ``#`` marks the end of an inner-line-comment I think.

Reply:
Oh I see!